infra/stacks/static-file-service/static-file-service.ts

```diff
@@ -10,13 +10,19 @@ import {
   serviceLoadBalancer,
 } from '../../packages/resources';
 import { EcrImage } from '../../packages/service';
-import { BASE_DOMAIN, MACRO_SUBDOMAIN_CERT } from '../../packages/shared';
+import {
+  BASE_DOMAIN,
+  getDomainName,
+  getServiceUrl,
+  MACRO_SUBDOMAIN_CERT,
+  ServiceUrl,
+} from '../../packages/shared';
 import { StaticFileCloudFront } from './distribution';
 
 const stack = pulumi.getStack();
 export const SERVICE_NAME = 'static-file-service';
+// This is used for the cloudfront distribution url
 export const SERVICE_DOMAIN_NAME = `static-file-service${stack === 'prod' ? '' : `-${stack}`}`;
```

Comment:
We are switching on 'stack' here to derive new string here. We should avoid that.